omreaderlib/src/Interface.scala

```diff
@@ -0,0 +1,61 @@
+package org.chipsalliance.t1.omreaderlib
+
+import scala.reflect.runtime.universe._
+import chisel3.panamalib.option._
+import chisel3.panamaom._
+import chisel3.panamaconverter.PanamaCIRCTConverter
+
+object OMReader {
+  def fromFile(mlirbcFile: os.Path, className: String): OMReader = {
+    new OMReader(os.read.bytes(mlirbcFile), className)
+  }
+
+  def fromBytes(mlirbc: Array[Byte], className: String): OMReader = {
+    new OMReader(mlirbc, className)
+  }
```

Comment:
We don't need `fromBytes` since in our case, we only need to read mlirbc into OMReader.
```suggestion
  private def fromBytes(mlirbc: Array[Byte], className: String): OMReader = {
    new OMReader(mlirbc, className)
  }  
```

Reply:
This is omreaderlib function, we read mlicbc in cli, then pass the bytes into lib.